src/containers/Cardblock/index.js

```diff
@@ -62,25 +65,25 @@ const Cardblock = (props) => {
             <div className="cardblock">
                 <CardList
                   checkboxMain={state.checked}
-                  cards={props.cards} />
+                  cards={cards} />
             </div>
-            <button type="text" onClick={() => props.onDeleteCards(props.cards)}>Удалить выбранные карточки</button>
-            <button type="text" onClick={() => props.onAddCard(props.cards)}>Создать карточку</button>
+            <button type="text" onClick={() => dispatch(actionCreators.deleteCards(cards))}>Удалить выбранные карточки</button>
+            <button type="text" onClick={() => dispatch(actionCreators.addCard(cards))}>Создать карточку</button>
         </div>
       );
 };
 
-const mapStatetoProps = state => {
-  return {
-    cards: state.cards
-  }
-}
+// const mapStatetoProps = state => {
+//   return {
+//     cards: state.cards
+//   }
+// }
 
-const mapDispatchToProps = dispatch => {
-  return {
-    onDeleteCards: (cards) => dispatch(actionCreators.deleteCards(cards)),
-    onAddCard: (cards) => dispatch(actionCreators.addCard(cards))
-  }
-}
+// const mapDispatchToProps = dispatch => {
+//   return {
+//     onDeleteCards: (cards) => dispatch(actionCreators.deleteCards(cards)),
+//     onAddCard: (cards) => dispatch(actionCreators.addCard(cards))
+//   }
+// }
```

Comment:
Ideally there should be no commented code. If you require any artifact code to be stored at some point - you can save it as a part of some intermediate commit. But final commit should have it removed.